ui/appui-react/src/test/TestUtils.ts

```diff
@@ -16,6 +16,8 @@ import {
   FrameworkState, SyncUiEventDispatcher, ToolSettingsManager, UiFramework,
 } from "../appui-react";
 import { TestContentControl } from "./frontstage/FrontstageTestUtils";
+import userEvent from "@testing-library/user-event";
+export {userEvent};
```

Comment:
why is this being exported and not just imported everywhere as needed?

Reply:
Because for some reason, which I guess is related to the way this library is declared, "userEvent" does not trigger the automatic import mechanism by intellisense in vscode, which means each time you want to use it, you need to go back to the top of the file and make the import yourself, which, over the course of 40 files, I found really annoying. Exporting it like that makes the import fixes and the auto import working correctly.

Reply:
Gotcha. I'm guessing vscode fails to auto import it due to it being a default and not named export